cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicationPolicyManager.java

```diff
@@ -20,16 +20,31 @@
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * Class to manage replication lifecycle events.
+ */
 public class ReplicationPolicyManager {
 
     private final List<Replicator> replicators = new ArrayList<Replicator>();
     private final ReplicationListener replicationListener = new ReplicationListener();
 
+    /**
+     * Interface with callback methods for replication lifecycle events.
```

Comment:
@brynh if the Android services don't use the manager, do you think the listener should be extracted into it's own class? Though we'll do that under another PR if so.